packages/config/docs/file_paths.md

```diff
@@ -0,0 +1,107 @@
+This document explains some of the main file paths used in a Netlify Site. This is mostly intended for code
+contributors.
+
+## Current directory
+
+Process's current directory, as specified by `process.cwd()`.
+
+This is mostly relevant for the Netlify CLI, where users can change it in their terminal.
+
+## Repository root
+
+Project root repository. In monorepos, this is the monorepo root.
+
+This is the closest parent with a `.git` directory. If none is found, this defaults to the current directory.
+
+## Build directory
+
+This is used as the current directory when running the build command and Build plugins.
+
+This is also used to resolve any file paths inside `netlify.toml` or the UI build settings, except for the `base`
+configuration property (which is relative to the repository root instead).
+
+We sometimes refer to this as the "base directory" or the "Site root" instead.
+
+Its value is (from highest to lowest priority):
+
+- The `base` property specified in `netlify.toml` or UI build settings. This is useful when targetting specific
+  directories inside a monorepo.
+- In the Netlify CLI: if a project has multiple Netlify Sites, the user can step into it inside a terminal (using `cd`
```

Comment:
> the user can step into it

I'm not sure exactly how to rephrase this since I'm a little unclear of the technical context, but I'd recommend using something other than a pronoun (it) to be a little clearer. 

Perhaps something like " In the Netlify CLI: If a project has multiple Netlify sites, the user can change directories to this build directory using the terminal."

Reply:
Makes sense, thanks!